### ISSUE TITLE
Chapter 11, page 224 - Real time updates not working

### ISSUE THREAD

**Issue by pmarangoni** [PacktPublishing/Web-Development-with-Blazor-Third-Edition#33]
Below #13:

> When we change the text of the blog post in the second window, the change should be reflected in real time in the first window.

Doesn't work. Changing the text does not affect the other browser.

**Comment by pmarangoni** [comment]
> Did you manage to understand where the problem is? Have you by any chance solved it?

No, I haven't. I gave up on this book a few weeks ago. It's pretty much a useless mess.

**Comment by zaytar68** [comment]
It works on the blog text. If you change the title it's like a new blogpost. Because title = id if i remember correctly.

**Comment by EngstromJimmy** [comment]
I pulled down the code from here, and I got it to work.

The idea is to open the blog post (non edit) and another tab where you edit the blog post. Then the view of the blog post should change.

If you try it with both server or webassembly, use the same for both Post.razor and Admin/BlogPostEdit.razor.


It is only the text (not the title) that triggers a change.
